src/ethereum_test_tools/common/types.py

```diff
@@ -982,6 +1041,12 @@ def default(self, obj):
                 b["blocknumber"] = str(obj.block_number)
             if obj.chain_name is not None:
                 b["chainname"] = obj.chain_name
+            if obj.txs is not None:
+                b["transactions"] = obj.txs
+            if obj.ommers is not None:
+                b["uncleHeaders"] = obj.ommers
+            if obj.withdrawals is not None:
+                b["withdrawals"] = obj.withdrawals
```

Comment:
Are we actually adding these fields during filling to the block object ?

Reply:
I have only added them within `make_blocks`  for `blockchain_test.py` so far, but plan to add them to the latter within `state_test.py` as well.